specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/dedicatedHost.json

```diff
@@ -757,6 +757,61 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}/hosts/{hostName}/redeploy": {
+      "post": {
+        "tags": [
+          "DedicatedHost"
+        ],
+        "operationId": "DedicatedHosts_Redeploy",
+        "description": "Redeploy the dedicated host. The operation will complete successfully once the dedicated host has migrated to a new node and is running. To determine the health of VMs deployed on the dedicated host after the redeploy check the Resource Health Center in the Azure Portal. Please refer to https://docs.microsoft.com/azure/service-health/resource-health-overview for more details.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hostGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the dedicated host group."
+          },
+          {
+            "name": "hostName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the dedicated host."
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
```

Comment:
whats he schema for this response?

Reply:
We are not returning any body in the response for this post action api how it can complete synchronously or asynchronously as a  long running operations and the client generally polls on the async operation header returned in the response header, this is standard Azure RPC as implemented for post api defined at https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/async-api-reference.md#call-action-post-asynchronously, I believe a response of 204 is returned when the resource does not exist primarily in context of a Delete call as per azure resource provider contract.

Reply:
@rkmanda

Reply:
You do not have to specify 200 in this case, also see https://github.com/Azure/azure-rest-api-specs/pull/26427/checks?check_run_id=18386335117 for errors

Reply:
Is the same true of restart above? This is the source of one of my lint errors.

Reply:
For restart, you can also only have 202, if there's no content